styles.css

```diff
@@ -336,8 +336,6 @@ background-color: white;
 
   .idea-cards-main-area {
     display: grid;
-    min-width: 150px;
-    width: auto;
```

Comment:
To who it may concern -I noticed that the Width was not exactly what we wanted it to be. I think this fixes it. Please review and tell me what you think and if there are any implications to changing the in-width or width I may have missed. Sincerely  -Taylor

Reply:
Looks great! I checked this in my console & it is working. Thank you for fixing it!

Reply:
Dearest Taylor, 
Have we looked into how these changes may affect other views or is this in the media query? Additionally, does this make our mobile view less responsive on any specific devices? 
Warmest Regards,
KJ

Reply:
In regards to your last message. I reflected on the questions you submitted and found that I am only changing the mobile query and it does not seem to be making anything less responsive.
-Thank you for getting back to me with your feedback
Taylor